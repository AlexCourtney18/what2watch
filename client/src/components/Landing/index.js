import React from 'react';
import landingImage from "../../assets/landing-image.jpg"

function Landing() {
    return (
        <section className='my-5'>
            <h1 id="title">What To Watch</h1>
            <img src={landingImage} className="my-2" style={{ width: "75%" }} alt="cover" />
        </section>
    );
}

export default Landing;