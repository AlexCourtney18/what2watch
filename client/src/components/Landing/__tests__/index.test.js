import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Landing from '..';
afterEach(cleanup);

describe('Landing component', () => {
    //first test
    it('renders', () => {
        render(<Landing />);
    });

    //second test
    it('matches snapshot DOM node structure', () => {
        const { asFragment } = render(<Landing />);
        expect(asFragment()).toMatchSnapshot();
    });
})