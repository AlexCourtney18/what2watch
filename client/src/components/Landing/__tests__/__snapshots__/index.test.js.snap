// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing component matches snapshot DOM node structure 1`] = `
<DocumentFragment>
  <section
    class="my-5"
  >
    <h1
      id="title"
    >
      What To Watch
    </h1>
    <img
      alt="cover"
      class="my-2"
      src="landing-image.jpg"
      style="width: 100%;"
    />
  </section>
</DocumentFragment>
`;
